Database/Corrections/TBC/tbcQuestFixes.lua

```diff
@@ -532,6 +532,9 @@ function QuestieTBCQuestFixes:Load()
         [9962] = {
             [questKeys.triggerEnd] = {"Brokentoe Defeated", {[zoneIDs.NAGRAND]={{43.32,20.72},},}},
         },
+        [9966] = {
+            [questKeys.requiredLevel] = {80},
+        },
```

Comment:
To mark a quest as "not in the game" simple add it to the existing list in [QuestieQuestBlacklist.lua](https://github.com/Questie/Questie/blob/c262d0fa2c9b10239307e801e6c3f77133ed6c1b/Database/Corrections/QuestieQuestBlacklist.lua)